cfme/automate/explorer.py

```diff
@@ -613,7 +613,10 @@ class InstanceFieldsRow(pretty.Pretty):
     Args:
         row_id: Sequential id of the row (begins with 0)
     """
-    table = Table("//div[@id='form_div']//table[@class='style3']")
+    # REVIEW: This locator seems wrong. The instance field table in 53z and
+    # upstream appears to be a SplitTable under div#inst_grid_div
+    # On an instance's page, div#form_div doesn't seem to exist.
+    table = Table("//div[@id='form_div']//table")
```

Comment:
@seandst I checked in instances, that the locator that works for the fields table is

```
//div[@id='form_div']/table[preceding-sibling::*[.='Fields']]
```